### PR TITLE
Aarch + osx

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
 
-cmake -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
+mkdir -p build
+pushd build
+cmake ${CMAKE_ARGS} ..
```
Comment (h-vetinari) [inline recipe/build.sh:6]: Probably a good idea to keep those
```suggestion

cmake ${CMAKE_ARGS} \
    -DCMAKE_INSTALL_PREFIX=$PREFIX \
    -DCMAKE_INSTALL_LIBDIR=lib \
    ..

```

File: recipe/build.sh [hunk 2]
```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
```
Comment (h-vetinari) [inline recipe/build.sh:1]:
```suggestion
#!/bin/bash
set -ex
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win and vc<14]
```
Comment (h-vetinari) [inline recipe/meta.yaml]: Obsolete
```suggestion
```

File: recipe/build.sh (hunk 1 above)
Comment (hmaarrfk) [reply recipe/build.sh:6]: they are defined in the CTNG activation feedstock it seems.

File: recipe/build.sh (hunk 2 above)
Comment (hmaarrfk) [reply recipe/build.sh:1]: thx. done

File: recipe/meta.yaml (hunk 3 above)
Comment (hmaarrfk) [reply recipe/meta.yaml]: thx. done

File: recipe/build.sh (hunk 1 above)
Comment (hmaarrfk) [reply recipe/build.sh:6]: https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/9661e4c851131527fab6a895ceb567b5172b9d48/recipe/activate-gcc.sh#L150

**Comment by h-vetinari** [comment]
I checked the package [content](https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Ftsl_robin_map-1.2.1-hf52228f_0.conda) and it's header-only, so we don't need a run-export.

**Comment by hmaarrfk** [comment]
good call!

**Pull request merged**